Sources/App/Models/EventReg/EventReg.generated.swift

```diff
@@ -57,9 +61,9 @@ extension EventReg: Preparation {
   static func prepare(_ database: Database) throws {
     try database.create(self) { builder in
       builder.id()
-      builder.foreignId(for: RegForm.self, optional: false, unique: false, foreignIdKey: Keys.regFormId)
-      builder.foreignId(for: User.self, optional: false, unique: false, foreignIdKey: Keys.userId)
-      builder.enum(Keys.status, options: ["approved","rejected","waiting","canceled"])
+      builder.foreignId(for: RegForm.self, optional: false, unique: false, foreignIdKey: Keys.regFormId, foreignKeyName: self.entity + "_" + Keys.regFormId)
+      builder.foreignId(for: User.self, optional: false, unique: false, foreignIdKey: Keys.userId, foreignKeyName: self.entity + "_" + Keys.userId)
+      builder.enum(Keys.status, options: ["approved", "notAppeared", "rejected", "waiting", "canceled"])
```

Comment:
Maybe `RegistrationStatus.something.rawValue` here? And we will have strings only in Registration Status enum, no duplicating and so on :)

Reply:
It's generated, so mistake no nave place  ))